docs/Meadow/Meadow.OS/Cellular/index.md

```diff
@@ -173,16 +217,67 @@ void CellAdapter_NetworkConnected(INetworkAdapter networkAdapter, INetworkAdapte
 }
 ```
 
-> **Notes**: Before checking the Cell Quality Signal (CSQ) and module IMEI, ensure a successful connection has been established. The CSQ is a static value (0-31) representing the signal quality obtained on the connection. To convert this number to dBm, you need to use the formula: dBm = -113 + CSQ * 2 (where CSQ is the returned value).
+> **Notes**: Before using the mentioned properties, ensure a successful connection has been established. The `Csq` property returns a static value (0-31) representing the signal quality obtained on the connection.
+
+### Fetching Cell Signal Quality
+
+It's important to note that the `Csq` property returns a cached value obtained from the connection, then to retrieve the most up-to-date CSQ (Cellular Signal Quality), you should utilize the `GetSignalQuality` method, as illustrated in the following example:
+
+```csharp
+var cell = Device.NetworkAdapters.Primary<ICellNetworkAdapter>();
+
+double csq  = cellAdapter.GetSignalQuality();
+Console.WriteLine("Cell Signal Quality: " + csq);
+```
+
+> **Notes**: To convert the CSQ value to dBm, you need to use the formula: dBm = -113 + CSQ * 2 (where CSQ is the returned value). You may experience disconnection from the cellular network for a brief period while the module gets the signal quality, so we suggest to avoid calling this method frequently to ensure a seamless user experience.
+
+# Using GNSS with cellular modules
```

Comment:
This section should be removed as well, since it's not present in the current OS version.